src/common/ormConnection.ts

```diff
@@ -32,12 +32,8 @@ export const getTypeORMConnection: () => Promise<Connection> = async () => {
     if (conn === undefined) {
         conn = createConnection({
             type: "postgres",
-            host: process.env.PGHOST,
-            port: parseInt(process.env.PGPORT || "5432"),
-            username: process.env.PGUSER,
-            password: process.env.PGPASSWORD,
-            database: process.env.PGDATABASE,
             synchronize: true,
+            extra: { ssl: { rejectUnauthorized: false } },
```

Comment:
Could we have this conditionally, similar to https://github.com/interlay/polkabtc-stats/blob/master/src/common/pool.ts#L4? To allow running on localhost.

Reply:
@theodorebugnet this applies only when `PGSSLMODE=require` so it will have no implications when running on localhost.